its/ruling/src/test/resources/expected/kubernetes/kubernetes-S1135.json

```diff
@@ -1,19 +1,4 @@
 {
-"kubernetes-project:ruling/src/test/resources/sources/kubernetes/helm/S5849/templates/deployment.yaml": [
-37
-],
-"kubernetes-project:ruling/src/test/resources/sources/kubernetes/helm/S6428/templates/deployment.yaml": [
-35
-],
-"kubernetes-project:ruling/src/test/resources/sources/kubernetes/helm/S6429/templates/deployment.yaml": [
-37
-],
-"kubernetes-project:ruling/src/test/resources/sources/kubernetes/helm/S6431/templates/deployment.yaml": [
-17
-],
-"kubernetes-project:ruling/src/test/resources/sources/kubernetes/helm/S6433/templates/deployment.yaml": [
-37
-],
```

Comment:
Missing issues are due to the fact that some functions (`default`, `toYaml`, `template`) are not supported, so template evaluation fails and files are skipped.
For `template` function to work properly, we would need to discover and load `*.tpl` files where templates are defined